### ISSUE TITLE
revert back change in DynamicAttributeService in 3.1

### ISSUE THREAD

**Issue by WeizhengSap** [SAP/spartacus#12100]
We added 2 functions in `DynamicAttributeService` in 3.1: `addAttributesToComponent`, and `addAttributesToSlot`. These 2 functions only work with our current SmartEdit lib, which is releaed in 3.2. We also added deprecation comments to `addDynamicAttributes` indicate that this function is deprecated since 3.2.

We need revert back the changes we made in `DynamicAttributeService` in 3.1.

**Comment by WeizhengSap** [comment]
I installed the lib using `ng add @spartacus/schematics@3.1`, and in `package.json`, we have version of `^3.1.2`.
![Screen Shot 2021-04-28 at 2 13 02 PM](https://user-images.githubusercontent.com/44440575/116447765-b6934980-a82e-11eb-896c-45d834bf171e.png)
I open this in `spartacus/core` and I can see this:
![Screen Shot 2021-04-28 at 2 14 36 PM](https://user-images.githubusercontent.com/44440575/116447847-cd39a080-a82e-11eb-939b-811dde3be902.png)
The `decorators` folder and the two functions `addAttributesToComponent` and `addAttributesToSlot` was added with SmartEdit lib, which should be released in 3.2.
Then I opened this:
![Screen Shot 2021-04-28 at 2 27 59 PM](https://user-images.githubusercontent.com/44440575/116447976-f823f480-a82e-11eb-84c7-13688a0be80d.png)
So, `@spartacus/core@^3.1.2` give us `@spartacus/core@3.2.0`.
